src/file/file.repository.ts

```diff
@@ -114,20 +114,13 @@ export default class FileRepository {
     });
   }
 
-  /**
-   * Retrieve the root folder (IFile) of a user by its name.
-   * @param folderName - the name of the folder.
-   */
-  static getRootFolder(folderName: string): Promise<IFile | null> {
-    return fileModel.findOne({ name: folderName, isRootFolder: true }).exec();
-  }
-
   /**
    * Retrieve a file residing in a folder by its name.
    * @param parentId - the folder id.
    * @param filename - the name of the file (should be unique in the folder).
    */
-  static getFileInFolderByName(parentId: string, filename: string): Promise<IFile | null> {
-    return fileModel.findOne({ name: filename, parent: new ObjectID(parentId), deleted: false }).exec();
+  static getFileInFolderByName(parentId: string, filename: string, ownerID: string): Promise<IFile | null> {
```

Comment:
`parentId` should be optional and be the last parameter of the function.

Reply:
there is no meaning to the function without parentId

Reply:
Making it optional is like making it default to `undefined` so it would be more fitting than giving it an empty string.
Now that I think about it, the type should be `string | null` since the root folder is now `null` so you would have to explicitly call the function with null and it wouldn't be an optional parameter.